### ISSUE TITLE
circRNA reads on IGV

### ISSUE THREAD

**Issue by gnilihzeux** [bioinfo-biols/CIRI-long#16]
Hi,
is there any option to save mapped reads? So, we could display the circRNA reads on IGV.

**Comment by Kevinzjy** [comment]
I am afraid not, the current version of CIRI-long does not generate reads alignment results.

**Comment by JinwenBioinfo** [comment]
So,  whether if I could generate those using your strategy in the paper ?
```
Cleaned reads were aligned to the mouse genome (GRCm38) version M20 (Ensembl 95) using minimap2 (ref. 45)
with the ‘–x splice’ option.
```
BTW, is it a better option using sequences in `xxx.cand_circ.fa` instead of cleaned reads ?

Thanks.

**Comment by gnilihzeux** [comment]
So,  whether if I could generate those using your strategy in the paper ?
```
Cleaned reads were aligned to the mouse genome (GRCm38) version M20 (Ensembl 95) using minimap2 (ref. 45)
with the ‘–x splice’ option.
```
BTW, is it a better option using sequences in `xxx.cand_circ.fa` instead of cleaned reads ?

Thanks.

**Comment by Kevinzjy** [comment]
Yes, you could use minimap2 to align cand_circ.fa to the reference genome. The result should be slightly different from the final collapsed output, but should still be enough for visualizing circRNA isoforms.

**Comment by gnilihzeux** [comment]
Well, it might be better if I just use reads in `xxx.reads` to align `xxx.cand_circ.fa` to the reference genome.
